src/picosvg/svg_transform.py

```diff
@@ -32,7 +32,7 @@
 )
 from picosvg.svg_meta import ntos
 
-DECOMPOSITION_ALMOST_EQUAL_TOLERANCE = 1e-6
+DECOMPOSITION_ALMOST_EQUAL_TOLERANCE = 1e-5
```

Comment:
I think even this is probably excessive, 1e-4 or even 3 is probably just fine

Reply:
changed to 1e-4